.github/workflows/branches-and-prs.yml

```diff
@@ -19,35 +19,38 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ['ubuntu-latest']
         variant: ['2.5', '3.0', '4.0']
-        java: ['8', '11', '17', '21']
+        java: ['8', '11', '17', '21', '22']
+        os: ['ubuntu-latest']
         exclude:
-          - os: 'ubuntu-latest'
-            variant: '2.5'
+          - variant: '2.5'
```

Comment:
Why did you re-order all the elements?

Reply:
Please re-read the PR description (with the screenshots):
> As a bonus, I also improved the visibility in GitHub Actions UI, the order of matrix variables has been changed (I might move it to a separate PR, if desired).
> 
> Before (JDK version and Groovy variant are not visible in the failing builds):

Reply:
got it